app/navigation/navigation-core/src/main/kotlin/com/hedvig/android/navigation/core/HedvigDeepLinkContainer.kt

```diff
@@ -49,6 +51,8 @@ internal class HedvigDeepLinkContainerImpl(
   override val contractWithoutContractId: String = "$baseDeepLinkDomain/contract"
   override val contract: String = "$baseDeepLinkDomain/contract?contractId={contractId}"
 
+  override val terminateInsurance: String = "$baseDeepLinkDomain/terminateInsurance?insuranceId={insuranceId}"
```

Comment:
Here this `insuranceId` also shows that it's different from the `contractId` right 2 lines above and I wonder if these two actually are referring to the same kind of id or not. Which is what makes me wonder if perhaps this should also be `contractId` if it's the case.